include/openPMD/IO/AbstractIOHandler.hpp

```diff
@@ -61,7 +61,11 @@ class unsupported_data_error : public std::runtime_error
  * @brief Determine what items should be flushed upon Series::flush()
  *
  */
+#ifdef __NVCOMPILER_MAJOR__
+enum class FlushLevel
+#else
 enum class FlushLevel : unsigned char
+#endif
```

Comment:
Is it worth adding this work-around instead of generally using `enum class` without a type?
```suggestion
enum class FlushLevelf
```

Reply:
We can also do that, but we should add a comment why we do it
```suggestion
// do not write `enum class FlushLevel : unsigned char` here since NVHPC
// does not compile it correctly
enum class FlushLevel
```